src/pages/OrgRegister.tsx

```diff
@@ -1,91 +1,116 @@
+                  >
+                  </textarea>
+                </div>
+                <div className='-mt-4 pr-[60%] md:pr-[38%] lg:pr-[50%]'>
+                  {errors.description && (<small className='text-red-600'>description is required</small>)}
+                </div>
+                <div className="flex w-80  md:w-[75%] justify-between rounded mb-5 mt-5">
+                  <label className="flex items-left text-xs dark:text-dark-text-fill ">
+                    <input
+                      type="checkbox"
+                      name="remember"
+                      className="mr-1 dark:text-dark-text-fill dark:border-white dark:bg-dark-frame-bg"
+                    />
+                    I agree to the Pulse's <span className='text-primary'>Terms & Conditions</span>
+                  </label>
+                </div>
+                <button
+                  type="submit"
+                  className="w-1/4 text-center mx-[20%] py-2 rounded-lg bg-primary text-white hover:bg-green-dark focus:outline-none"
+                >
+                  Register
+                </button>
+              </div>
+            </form>
           </div>
-          <button
-            type="submit"
-            className="w-1/2 lg:w-1/4 text-center lg:mx-40 py-3 mt-9 rounded bg-primary text-white hover:bg-green-dark focus:outline-none my-1"
-          >
-            Register
-          </button>
         </div>
-        <div className='hidden bg-primary dark:bg-primary px-6 h-screen lg:flex lg:flex-col pt-24 lg:mt-32 lg:mr-24 lg:mb-16 rounded-tr-2xl rounded-br-2xl 2xl:mt-32 '>
-          <div className='flex items-center justify-center text-white pb-14 lg:pb-6'>
-            <h1 className="text-2xl 2xl:text-4xl 2xl:py-10 2xl:mb-10 font-bold px-2 uppercase text-white dark:text-dark-text-fill font-sans">
-              Get started
-            </h1>
-          </div>
-          <h2 className='text-2xl font-bold text-dark-text-fill text-left px-5 font-sans'>Welcome to the page where your management starts</h2>
-          <p className='max-w-[600px] text-start text-xl text-white my-5 px-5'>
-          With your own organization, you get a custom domain where you can manage your trainees/students and you can handle all processes related to your program.
-            </p>
-          <Link to="get-started" className='border-b-1 font-bold text-white px-5'>Explore docs</Link>
-          <div>
-            <h3 className='font-bold text-white text-lg mt-5 mb-2 px-5'> Your organization link</h3>
-            <button className='rounded bg-white text-primary font-bold py-2 px-8 ml-4 mt-3'>{name.replaceAll(" ", "-").toLowerCase()}<span>.pulse.org</span></button>
-          </div>
+        <div className="md:w-2/5 bg-primary text-white lg:rounded-tr-2xl lg:rounded-br-2xl md:py-14 md:px-2 sm:w-full dark:rounded-none sm:rounded-none sm:px-12 sm:py-4 md:block sm:hidden dark:text-dark-text-fill ">
```

Comment:
please add the corner-rounding even in dark-mode

![image](https://user-images.githubusercontent.com/28519395/179767821-8871a318-c094-4cd7-b458-586a4810dff7.png)